Extension/tslint.json

```diff
@@ -1,31 +1,184 @@
+        "no-return-await": true,
+        "no-shadowed-variable": false,
+        "no-sparse-arrays": true,
+        "no-string-literal": false,
+        "no-string-throw": false,
+        "no-submodule-imports": false,
+        "no-switch-case-fall-through": true,
+        "no-this-assignment": true,
+        "no-trailing-whitespace": true,
+        "no-unbound-method": false,
+        "no-unnecessary-callback-wrapper": true,
+        "no-unnecessary-class": false,
+        "no-unnecessary-initializer": true,
+        "no-unnecessary-qualifier": true,
+        "no-unnecessary-type-assertion": false,
+        "no-unsafe-any": false,
+        "no-unsafe-finally": true,
+        "no-unused-expression": true,
         "no-unused-variable": true,
+        "no-use-before-declare": false,
         "no-var-keyword": true,
-        "one-line": [true, "check-catch", "check-finally", "check-else", "check-open-brace", "check-whitespace"],
+        "no-var-requires": true,
+        "no-void-expression": false,
+        "number-literal-format": true,
+        "object-literal-key-quotes": false,
+        "object-literal-shorthand": false,
+        "object-literal-sort-keys": false,
+        "one-line": [
+            true,
+            "check-catch",
+            "check-finally",
+            "check-else",
+            "check-open-brace",
+            "check-whitespace"
+        ],
+        "one-variable-per-declaration": true,
+        "only-arrow-functions": false,
+        "ordered-imports": false,
+        "prefer-conditional-expression": false,
+        "prefer-const": false,
+        "prefer-for-of": false,
+        "prefer-function-over-method": false,
+        "prefer-method-signature": true,
+        "prefer-object-spread": true,
+        "prefer-readonly": false,
+        "prefer-switch": false,
+        "prefer-template": false,
+        "prefer-while": true,
+        "promise-function-async": false,
         "promise-must-complete": true,
+        "quotemark": false,
+        "radix": false,
+        "restrict-plus-operands": false,
+        "return-undefined": false,
         "semicolon": true,
+        "space-before-function-paren": false,
+        "space-within-parens": true,
+        "strict-boolean-expressions": false,
+        "strict-type-predicates": true,
+        "switch-default": false,
+        "switch-final-break": false,
+        "trailing-comma": true,
         "triple-equals": true,
         "type-literal-delimiter": true,
-        "typedef": [true, "variable-declaration", "call-signature"],
-        "whitespace": [true, "check-branch", "check-operator", "check-separator", "check-preblock", "check-type"]
+        "typedef": [
+            true,
+            "variable-declaration",
+            "call-signature"
+        ],
+        "typedef-whitespace": true,
+        "typeof-compare": true,
+        "unified-signatures": true,
+        "use-default-type-parameter": true,
+        "use-isnan": true,
+        "variable-name": false,
```

Comment:
Delete this line. This should be handled by `ext-variable-name`

Reply:
What is ext-variable-name? It doesn't appear to be supported by my TsLint extension.

Reply:
Part of [vrsource-tslint-rules](https://github.com/vrsource/vrsource-tslint-rules#ext-variable-name).